.gitignore

```diff
@@ -21,6 +21,7 @@ Move.lock
 !crates/sui-framework/packages/move-stdlib/Move.lock
 !crates/sui-framework/packages/sui-framework/Move.lock
 !crates/sui-framework/packages/sui-system/Move.lock
+!crates/sui-framework/packages/deepbook/Move.lock
```

Comment:
we track other packages in `sui-framework` so `deepbook` should be included as well